config/zones/IE.yaml

```diff
@@ -48,13 +48,19 @@ emissionFactors:
       source: BEIS 2021
       value: 0.0
     coal:
-      datetime: '2021-01-01'
-      source: EU-ETS, ENTSO-E 2021
-      value: 758.4409086
+      - datetime: '2021-01-01'
+        source: EU-ETS, ENTSO-E 2021
+        value: 758.4409086
+      - datetime: '2022-01-01'
+        source: EU-ETS, ENTSO-E 2022
+        value: 1082.54
     gas:
-      datetime: '2021-01-01'
-      source: EU-ETS, ENTSO-E 2021
-      value: 396.8903896
+      - datetime: '2021-01-01'
+        source: EU-ETS, ENTSO-E 2021
+        value: 396.8903896
+      - datetime: '2022-01-01'
+        source: EU-ETS, ENTSO-E 2022
+        value: 624.48
```

Comment:
a bit high, we can consider revising the upper limit for gas EF (right now at 1700 g/kWh)